packages/host/app/routes/card.ts

```diff
@@ -38,11 +41,15 @@ export default class RenderCard extends Route<
     let url = path
       ? new URL(`/${path}`, ownRealmURL)
       : new URL('./', ownRealmURL);
+    await this.localRealm.startedUp;
     try {
       let instance = await this.cardService.loadModel(url);
       return instance.constructor.getComponent(instance, 'isolated');
     } catch (e) {
       (e as any).failureLoadingIndexCard = url.href === ownRealmURL;
+      if (isLocalRealm && !this.localRealm.isAvailable) {
+        return LocalRealmNotConnectedComponent;
```

Comment:
I think this should perhaps return a component instance? I don't know how to create one programmatically, though.

Reply:
you can turn this module into a .gts file and just include your component right here, instead of importing it, if that's what you mean.
```ts
class LocalRealmNotConnectedComponent extends Component {
  <template>
    LocalRealmNotConnectedComponent: Local realm not connected.
  </template>
}
```

Reply:
(a "component instance" is a bit of a misnomer. If you actually look at the types used for returning components it's a `ComponentLike<T>` type that is essentially a class prototype)